app/assets/stylesheets/privacy.scss

```diff
@@ -0,0 +1,13 @@
+.privacy-policy {
+  h1 {
+    font-family: Pathway Gothic One;
```

Comment:
Is this an exception to the rule that all Pathway Gothic One fonts should be uppercase?

Reply:
ah, good catch - it is supposed to be uppercase. i'll see if i can generalize this rule for h1s, and if not, will add a `text-transform` here.